platform/android/java/lib/src/org/godotengine/godot/input/GodotInputHandler.java

```diff
@@ -433,7 +433,7 @@ static boolean isMouseEvent(MotionEvent event) {
 	}
 
 	private static boolean isMouseEvent(int eventSource) {
-		boolean mouseSource = ((eventSource & InputDevice.SOURCE_MOUSE) == InputDevice.SOURCE_MOUSE) || ((eventSource & (InputDevice.SOURCE_TOUCHSCREEN | InputDevice.SOURCE_STYLUS)) == InputDevice.SOURCE_STYLUS);
```

Comment:
What's the reason for removing the `SOURCE_TOUCHSCREEN` check?

Reply:
I couldn't get it to work without removing that. I think `SOURCE_TOUCHSCREEN` is used for fingers also?

Reply:
Should we try it like this, or investigate further?

Reply:
From what I can test, there seems to be no regression in touch handling when removing that `SOURCE_TOUCHSCREEN`, I think it should be fine to try it like this.

Reply:
It looks like this change has introduced the following regression: #88621